spk/nano/Makefile

```diff
@@ -0,0 +1,25 @@
+SPK_NAME = nano
+SPK_VERS = 2.4.0
+SPK_REV = 1
+SPK_ICON = src/nano.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Nano is a text editor for Unix-like computing systems or operating environments using a command line interface.
```

Comment:
Can I suggest to be consistent with the naming? It's called `Nano` in the description, but the display names and changelog use `GNU Nano`. Perhaps simply using `nano` (lowercase, as per the website) would make the most sense?